### PR TITLE
Setup Tailwindcss

### PR DESCRIPTION
**Pull request by IcedTea2K**
- Don't forget to run `npm install` to see the changes.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{js,jsx,ts,tsx}"],
+  theme: {
+    extend: {
+      fontFamily: {
+        'inter' : ['Inter', 'sans-serif'],
```
Comment (Kevin-Tan25) [inline tailwind.config.js:7]: our font family is poppins for the main website could you please change the font to reflect that?

Comment (IcedTea2K) [reply tailwind.config.js:7]: Ah okay, I thought it was Inter because the new design for the website is in Inter.

Comment (IcedTea2K) [reply tailwind.config.js:7]: @Kevin-Tan25 Just added the fonts

**Review by Kevin-Tan25 — APPROVED** [review]
LGTM

**Pull request merged**